content/docs/hooks-state.md

```diff
@@ -130,13 +130,13 @@ function Example() {
   const [count, setCount] = useState(0);
 ```
 
-**What does calling `useState` do?** It declares a "state variable". Our variable is called `count` but we could call it anything else, like `banana`. This is a way to "preserve" some values between the function calls — `useState` is a new way to use the exact same capabilities that `this.state` provides in a class. Normally, variables "disappear" when the function exits but state variables are preserved by React.
+**कॉलिंग `useState` क्या करता है?** यह एक "state वेरिएबल" घोषित करता है। हमारे वेरिएबल को `count` कहा जाता है लेकिन हम इसे कुछ और भी कह सकते हैं जैसे की `banana`। यह फ़ंक्शन कॉल के बीच कुछ वैल्यूज को "संरक्षित" करने का एक तरीका है - `useState` एक class में `this.state` दवारा प्रदान की क्षमताओं को उपयोग करने का एक नया तरीका है। आम तौर पर, फ़ंक्शन के ख़तम होने पर वेरिएबल्स "गायब" हो जाते हैं, लेकिन state वेरिएबल्स React द्वारा संरक्षित होते हैं।
 
-**What do we pass to `useState` as an argument?** The only argument to the `useState()` Hook is the initial state. Unlike with classes, the state doesn't have to be an object. We can keep a number or a string if that's all we need. In our example, we just want a number for how many times the user clicked, so pass `0` as initial state for our variable. (If we wanted to store two different values in state, we would call `useState()` twice.)
+**हम `useState` को एक आर्गुमेंट के रूप में क्या पास करते हैं?** `useState()` Hook के लिए initial state एकमात्र आर्गुमेंट है। classes के विपरीत, state को एक object होना आवश्यक नहीं है। अगर हमें आवश्यकता है तो हम एक संख्या या एक स्ट्रिंग रख सकते हैं। हमारे उदाहरण में, हम केवल एक संख्या चाहते हैं कि कितनी बार उपयोगकर्ता ने क्लिक किया, इसलिए हमारे वेरिएबल के लिए initial state के रूप में `0` पास करें। (यदि हम state में दो अलग-अलग वैल्यूज को संग्रहीत करना चाहते हैं, तो हम दो बार `useState()` का उपयोग करेंगे।)
```

Comment:
`अगर हमें आवश्यकता है तो हम एक संख्या या एक स्ट्रिंग रख सकते हैं।` => `हम एक संख्या या एक स्ट्रिंग रख सकते हैं अगर हमें सिर्फ ऐसी की आवश्यकता है तो।`

how about this?